README.md

```diff
@@ -0,0 +1,6 @@
+#Short instruction how to run weltyt.py in Docker:
+- In Terminal enter into this directory where files are.
+- Build an image to Docker by command: 'docker build -t weltyt .'
+- Run image by command: 'docker run -it weltyt'
+- If you have problem with permission try run it as root.
+- If you would like avoid permanent running as root, follow this instruction please: (https://github.com/sindresorhus/guides/blob/main/docker-without-sudo.md)
```

Comment:
I think we agreed to add an instruction how to run the script outside of the Docker also.